src/main/java/edu/CountriesWithVatNumber.java

```diff
@@ -0,0 +1,10 @@
+package edu;
+
+public interface CountriesWithVatNumber {
```

Comment:
No need an interface. A simple object with values is enough. In fact, enum fits the best in this case with a bunch of premade objects.